geo_json_metrics/gps_metrics.py

```diff
@@ -228,20 +228,29 @@ def calculate_metrics(df: pd.DataFrame) -> tuple[float, float, float]:
     return (median_avg, median_min, median_max)
 
 
+def verify_outlier(filtered_df: pd.DataFrame):
+    """Used to verify the outlier from segment_10240935_linestring.json.
+    The speed calculated was initially wrong. But now fixed.
```

Comment:
Can this be made more generic? It seems out-of-nowhere that specifically this line string has a specific outlier.

Reply:
done